theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -22,21 +24,64 @@ let fundingAgenciesApp = new Vue({
     deleteUrl: "", // Django endpoint to call for deleting a funder
     currentlyDeleting: {}, // store the funder that we are deleting
   },
+  mounted() {
+    if (this.resourceMode === "Edit") {
```

Comment:
Are we also checking somewhere that the resource is a published resource?

Reply:
Good call thanks!